### PR TITLE
Updating readme to have cmake fetchcontent example

### PR DESCRIPTION
**Pull request by cjappl**
Hi @cameron314 ,

Thanks for your work on this. Just saved me a lot of time doing an experiment.

I wrote a small addition to the README explaining how someone can easily add this target to existing CMake infrastructure. Figured it may be helpful to some.

### CODE REVIEW

**Comment by cameron314** [comment]
Thanks for this! It looks like a useful addition. Could you just adjust the new `#` headings to `##`, and new `##` headings to `###`?

Also, sorry for the delay in getting back to you.

File: README.md [hunk 1]
```diff
@@ -115,7 +116,37 @@ q.wait_dequeue_timed(number, std::chrono::milliseconds(10));
...
+)
+
+FetchContent_MakeAvailable(readerwriterqueue)
+
+add_library(my_target main.cpp)
+target_link_libraries(my_target PUBLIC readerwriterqueue)
+```
+
+In main.cpp:
+```
```
Comment (cameron314) [inline README.md:140]:
```suggestion
```cpp
```

File: README.md [hunk 2]
```diff
@@ -115,7 +116,37 @@ q.wait_dequeue_timed(number, std::chrono::milliseconds(10));
 ```
 
 
-## CMake installation
+# CMake
```
Comment (cameron314) [inline README.md:119]:
```suggestion
## CMake
```

File: README.md [hunk 3]
```diff
@@ -115,7 +116,37 @@ q.wait_dequeue_timed(number, std::chrono::milliseconds(10));
...
+## Using targets in your project
```
Comment (cameron314) [inline README.md:120]:
```suggestion
### Using targets in your project
```

File: README.md [hunk 4]
```diff
@@ -115,7 +116,37 @@ q.wait_dequeue_timed(number, std::chrono::milliseconds(10));
...
+```
+#include <readerwriterqueue.h>
+
+int main(void)
+{
+    moodycamel::ReaderWriterQueue<int> q(100);
+}
+```
+
+## Installing into system directories
```
Comment (cameron314) [inline README.md:149]:
```suggestion
### Installing into system directories
```

File: README.md [hunk 5]
```diff
@@ -115,7 +116,37 @@ q.wait_dequeue_timed(number, std::chrono::milliseconds(10));
...
+
+add_library(my_target main.cpp)
+target_link_libraries(my_target PUBLIC readerwriterqueue)
+```
+
+In main.cpp:
+```
+#include <readerwriterqueue.h>
+
+int main(void)
```
Comment (cameron314) [inline README.md:143]:
```suggestion
int main()
```

**Comment by cjappl** [comment]
Here you go, as requested! No problem in the delay :)

**Pull request merged**